analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -379,7 +378,8 @@ private void updateAllExtraGradleFilesWithGmeRepos() throws IOException, Manipul
         if (!gradleScriptsDirectory.exists()) {
             return;
         }
-        final Collection<File> extraGradleScripts = FileUtils.listFiles(gradleScriptsDirectory, new SuffixFileFilter(".gradle"),
+        final Collection<File> extraGradleScripts = FileUtils.listFiles(gradleScriptsDirectory,
+                FileFilterUtils.and(new SuffixFileFilter(".gradle"), new NotFileFilter(new NameFileFilter(GME_REPOS))),
```

Comment:
Proposed fixed is to exclude any existing `GME_REPOS` file from update as it will be overwritten later anyway.